tests/server_test.go

```diff
@@ -10368,7 +10369,7 @@ func runFluxBuiltinTest(t *testing.T, file *ast.File, u *url.URL, bucket string,
 
 	inspectCalls := stdlib.TestingInspectCalls(pkg)
 	if len(inspectCalls.Body) == 0 {
-		t.Skip("No tests in builtin test package")
+		t.Skip("No tests in init test package")
```

Comment:
Do we still need the EndToEnd tests built in if we're running externally?

Reply:
The Flux team said yes (for now), because they haven't fully reproduced the existing test-cases yet. I'll make a ticket for eventually deleting EndToEnd